app/serializers/search/user_serializer.rb

```diff
@@ -1,5 +1,8 @@
 module Search
   class UserSerializer < ApplicationSerializer
+    CUSTOM_ATTRIBUTES = "custom_attributes".freeze
```

Comment:
I decided to freeze this so we don't re-allocate the key every time we serialize a user for search.